src/hydra/parseHydraDocumentation.test.js

```diff
@@ -1276,33 +1288,55 @@ test('Parse entrypoint class without "supportedProperty" key', () => {
 
   fetch.mockResponses([entrypoint, init], [docs, init]);
 
-  parseHydraDocumentation("http://localhost/").catch(data => {
-    expect(data.message).toBe(
-      'The entrypoint definition has no "http://www.w3.org/ns/hydra/core#supportedProperty" key or it is not an array.'
-    );
-  });
+  const expectedError = { message: "" };
+
+  try {
+    await parseHydraDocumentation("http://localhost/");
+  } catch (error) {
+    expectedError.message = error.message;
+  }
+
+  expect(expectedError.message).toBe(
+    'The entrypoint definition has no "http://www.w3.org/ns/hydra/core#supportedProperty" key or it is not an array.'
+  );
 });
 
-test("Invalid docs JSON", () => {
+test("Invalid docs JSON", async () => {
   const docs = `{foo,}`;
 
   fetch.mockResponses([entrypoint, init], [docs, init]);
 
-  parseHydraDocumentation("http://localhost/").catch(data => {
-    expect(data).toHaveProperty("api");
-    expect(data).toHaveProperty("response");
-    expect(data).toHaveProperty("status");
-  });
+  const expectedError = {};
+
+  try {
+    await parseHydraDocumentation("http://localhost/");
+  } catch (error) {
+    expectedError.api = error.api;
```

Comment:
Change the value of `expectedError`: it will be easier than create new properties.

```js
expectedError = error
```